src/components/Navbar.jsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { Link } from "react-router-dom";
 import "../../src/App.css";
```

Comment:
You can consider using absolute imports it needs some change in configuration but its just a suggestion.